demos/asp-net-core/Divergent.CompositionGateway/Startup.cs

```diff
@@ -8,15 +8,12 @@ namespace Divergent.CompositionGateway
 {
     public class Startup
     {
-        public void ConfigureServices(IServiceCollection services)
-        {
-            services.AddRouting();
-            services.AddViewModelComposition();
-        }
+        public void ConfigureServices(IServiceCollection services) =>
+            services
+                .AddRouting()
+                .AddViewModelComposition();
```

Comment:
This was created as the original VS/new project template, I don't think we should change it, it'll only raise time wasting questions